modules/virt-preparing-nodes-and-cluster-pci.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+
+[id="virt-preparing-nodes-and-cluster-pci_{context}"]
+= Preparing nodes and the cluster
+
+To configure a hardware device as a host device for PCI passthrough, create a `MachineConfig` object file and add kernel arguments to enable Input-Output Memory Management Unit (IOMMU) for the device. Bind the PCI device to the Virtual Function I/O (VFIO) driver by creating a `MachineConfig` object file. Then, add the PCI device to the HyperConverged Operator.
```

Comment:
Let's remove the `for the device` since this kernel parameter is being configured per host not per device.

Reply:
Updated the content.